### ISSUE TITLE
pa11y docs accessibility test breaking due to GitHub Actions failing to run Chrome

### ISSUE THREAD

**Issue by micahellison** [jrnl-org/jrnl#2048]
### Diagnostic output

n/a

### Current Behavior

Actions log:
```
Poe => 'tasks:run_shell('"'"'npx pa11y-ci -c config.json'"'"')'
/home/runner/work/jrnl/jrnl/node_modules/puppeteer-core/lib/cjs/puppeteer/node/BrowserLauncher.js:312
                    throw new Error(`Could not find Chrome (ver. ${this.puppeteer.browserVersion}). This can occur if either\n` +
                          ^

Error: Could not find Chrome (ver. 138.0.7204.168). This can occur if either
 1. you did not perform an installation before running the script (e.g. `npx puppeteer browsers install chrome`) or
 2. your cache path is incorrectly configured (which is: /home/runner/.cache/puppeteer).
For (2), check out our guide on configuring puppeteer at https://pptr.dev/guides/configuration.
    at ChromeLauncher.resolveExecutablePath (/home/runner/work/jrnl/jrnl/node_modules/puppeteer-core/lib/cjs/puppeteer/node/BrowserLauncher.js:312:27)
    at ChromeLauncher.computeLaunchArguments (/home/runner/work/jrnl/jrnl/node_modules/puppeteer-core/lib/cjs/puppeteer/node/ChromeLauncher.js:93:24)
    at async ChromeLauncher.launch (/home/runner/work/jrnl/jrnl/node_modules/puppeteer-core/lib/cjs/puppeteer/node/BrowserLauncher.js:84:28)
    at async /home/runner/work/jrnl/jrnl/node_modules/pa11y-ci/lib/pa11y-ci.js:58:18

Node.js v20.20.0
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/runner/work/jrnl/jrnl/tasks.py", line 24, in run_shell
    subprocess.run(command, check=True, shell=True)
  File "/opt/hostedtoolcache/Python/3.11.14/x64/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command 'npx pa11y-ci -c config.json' returned non-zero exit status 1.
```

### Expected Behavior

n/a

### Repro Steps

This only happens in GitHub Actions AFAIK. Works fine for me locally.

### Debug output

n/a

### Other Information

This seems to be a common problem but I can't find a common solution. Verified we're already using the infamous `--no-sandbox` arg in `chromeLaunchConfig`. Some Googling suggests using additional GitHub Actions, which I don't particularly like (the less complexity in a closed runner like Actions, the better), but that might be the only resolution.

**Comment by micahellison** [comment]
Every time I bust the npm cache, it works, but if I re-run the job without changing anything, it fails.
